tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -33,7 +33,7 @@
 from tensorflow.python.ops import state_ops
 from tensorflow.python.ops import variable_scope as vs
 from tensorflow.python.training import saver
-from tensorflow.python.training.checkpointable import tracking as checkpointable_lib
+from tensorflow.python.training.tracking import tracking as trackable_lib
```

Comment:
remove?

Reply:
I merged from a previous commit bd36b48c555b2d46c41a179ed9f27a04806e9e66 for this change to solve the conflict.